client-samples/java/rest/src/test/java/com/ms/infra/example/application/TestMicroprofileConfigServiceShould.java

```diff
@@ -0,0 +1,69 @@
+package com.ms.infra.example.application;
+
+import com.ms.infra.example.application.config.MicroprofileConfigService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
Pedantic, but could you please remove the wildcard import? You can get IntelliJ to auto-format this away.